Makefile

```diff
@@ -9,7 +9,15 @@ help:
 
 .PHONY: services
 services:
+ifneq ("$(wildcard ./20190805_dump.sql.gz)","")
```

Comment:
probably best to make this a recipe unto itself. Make will then auto resolve it by running the recipe if the file doesn't exist.

Reply:
```diff
diff --git a/Makefile b/Makefile
index 0eb1957..99b5721 100644
--- a/Makefile
+++ b/Makefile
@@ -7,16 +7,14 @@ help:
        @echo "make services          Run the services that Concourse and the pipeline requires"
        @echo "make sql               Connect to the cnx-db database with a psql shell"
 
-.PHONY: services
-services:
-ifneq ("$(wildcard ./20190805_dump.sql.gz)","")
-       docker-compose up -d
-else
+dump.sql.gz:
        @echo Downloading db dump file first...
-       rsync -tv --progress -e 'ssh -T -c aes128-ctr -o Compression=no -x' backup1.cnx.org:/var/backups/db_dump/20190805_dump.sql.gz ./
-       docker-compose up -d
+       rsync -tv --progress -e 'ssh -T -c aes128-ctr -o Compression=no -x' backup1.cnx.org:/var/backups/db_dump/20190805_dump.sql.gz ./dump.sql.gz
        @echo Please wait ~2-3 minutes on first run because the DB dump will be restored now...
-endif
+
+.PHONY: services
+services: dump.sql.gz
+       docker-compose up -d
 
 
 .PHONY: sql
```